modules/ossm-cr-threescale.adoc

```diff
@@ -7,29 +7,31 @@
 
 = 3scale configuration
 
-Here is an example that illustrates the 3scale Istio Adapter parameters for the {ProductName} custom resource and a description of the available parameters with appropriate values.
+The following table explains the parameters for the 3scale Istio Adapter in the `ServiceMeshControlPlane` resource.
```

Comment:
It's not an insurmountable obstacle for customers to scroll to the table, but it's pretty cheap to move this para after the YAML and add a one-liner to introduce the YAML.

Reply:
I agree with this and everything else, @mikemckiernan . We have a work item to work on this entire assembly, but it's out of scope for me. I really can't fix it :/.